app/controllers/users.rb

```diff
@@ -1,8 +1,26 @@
 get '/signup' do
-  # if user is logged in, redirect, else
   erb :'users/signup'
 end
 
+get '/login' do
+  erb :'users/login'
+end
+
+post '/login' do
+  @user = User.find_by(username: params[:username])
+
+  if @user.nil?
```

Comment:
Sam is going to call a code smell on line 12. Instead of 'if @user.nil?' he would want it to be 'unless @user'